packages/blaze/view.js

```diff
@@ -174,18 +174,27 @@ Blaze.View.prototype.autorun = function (f, _inViewScope) {
     throw new Error("Can't call View#autorun from a Tracker Computation; try calling it from the created or rendered callback");
   }
 
-  var templateInstanceFunc = Blaze.Template._currentTemplateInstanceFunc;
+  // Use a local dictionary for locals to allocate them on heap. Helps to avoid
+  // the stack overflow.
```

Comment:
Yeah, write a short essay describing it to someone who's never thought of this before. ("Beginner's mind")